api/lib/application/preHandlers/feature-toggles.js

```diff
@@ -2,13 +2,6 @@ const config = require('../../config');
 const { NotFoundError } = require('../../application/http-errors');
 
 module.exports = {
-  async isCertifPrescriptionSCOEnabled() {
-    if (!config.featureToggles.certifPrescriptionSco) {
```

Comment:
Note for self: On avait des pre-handler pour les toggles... Penser à les faire ici plutôt que le check qui vient d'être crée dans les security pre-handler